docs/cloud/networks.md

```diff
@@ -0,0 +1,38 @@
+---
+sidebar_position: 2
+--- 
+
+
+# Networks
+
+Dragonfly Cloud lets you create *private networks* for your data stores. After you create a private network you can:
+
+ - Create a VPC *peering connection* to connect the Dragonfly Cloud private network with a VPC in your cloud account to establish communication between the two networks over private ip space. For more details see [Peering Connections](./connections).  
```

Comment:
Dragonfly Cloud allows you to establish a Virtual Private Cloud (VPC) to ensure that your Dragonfly data stores are accessible only within your network. This improves security, reduces network latency and saves on data transfer costs.

STEPS:
1. Create a Dragonfly VPC - In this step you create a VPC where your Dragonfly data store will be hosted. Data stores hosted in this VPC will be accessible only within this network which reduces the potential attack surface.
2. Peer your private Dragonfly VPC with your application VPC - in this step you will establish a connection between your application private cloud to your private Dragonfly VPC you created in step 1. This will allow your application to access your data stores.
3. Create private data stores - Private data stores are data stores that are available only inside your VPC and are isolated from other networks. 

Quota:
VPC are virtual resources created on the cloud. Each Cloud account is limited up to 2 VPCs and up to 4 peering connections. If your team requires more than that, contact support@dragonflydb.io to ask for a quota change.

Reply:
We use the term private netowrk in the prodcut, this is not a step by step guide for how to create a private endpoint data store, we can add such (but not in this pr)